### PR TITLE
Removed official summary results

### LINKED ISSUE #2: Remove "summary score".
Individual application benchmarks that industry can see and compare is the key value of MLPerf.  Summary score tries to go beyond that and compress complexity of AI into a scalar.  The example of the modern GPU industry (rapid innovation driven by per-application benchmarks, no summary score) indicates that the case for a summary score helping innovation is weak.  NVIDIA's counterargument is that it is a negative - detracts from innovation by discouraging interest in diverse AI workloads, and very subjective.  See https://www.spec.org/gwpg/gpc.data/vp12.1/summary.html and https://www.spec.org/gwpg/gpc.static/vp12.1info.html for an example of application benchmarking in SPEC taking a per-application benchmark approach.

Supporting orgs: NVIDIA, Intel, dividiti, Wave Computing, Cisco*, SambaNova

### PR DESCRIPTION
**Pull request by petermattson**
Fixes #2 Remove Summary Score

### CODE REVIEW

File: training_rules.adoc [hunk 1]
```diff
@@ -43,7 +43,11 @@ A _reference result_ is the median of five run results provided by the MLPerf or
 
 A _benchmark result_ is the median of five run results normalized to the reference result for that benchmark. Normalization is of the form (reference result / benchmark result) such that a better benchmark result produces a higher number.
 
-A _suite result_ is the geometric mean of the benchmark results for a suite implementation under the rules of a division.
+A _submission result set_ is a one benchmark result for each benchmark implementation in a  submission implementation set.
```
Comment (nvpaulius) [inline training_rules.adoc:46]: Do we mean that it's a "... is a set of results, one for each benchmark in a submission implementation set"?

**Comment by petermattson** [comment]
Exactly, will change language.

File: training_rules.adoc [hunk 2]
```diff
@@ -52,9 +56,7 @@ The following rules apply to all benchmark implementations.
 Benchmarking should be conducted to measure the framework and system performance as fairly as possible. Ethics and reputation matter.
 
 === System and framework must be consistent
-The same system and framework must be used for a suite result or group of benchmark results reported in a single context.
-
-Note that the reference implementations use different frameworks and hence cannot be used collectively for a valid suite result.
+The same system and framework must be used for a submission result set. Note that the reference implementations do not all use the same framework in v0.5.
 
 === System and framework must be available
 If you are measuring the performance of a publicly available and widely-used system or framework, you must use publicly available and widely-used used versions of the system or framework.
```
Comment (nvpaulius) [inline training_rules.adoc:62]: "widely used" could be interpreted in different ways, we should have one of the working groups define what we mean.

**Comment by petermattson** [comment]
Agreed. This whole section needs cleanup. However, beyond the scope of this
update?

**Pull request merged**